### PR TITLE
Merge v3.3.x into master

### PR DESCRIPTION
**Pull request by QuLogic**
## PR Checklist

- [x] Has pytest style unit tests (and `pytest` passes).
- [x] Is [Flake 8](https://flake8.pycqa.org/en/latest/) compliant (run `flake8` on changed files to check).
- [n/a] New features are documented, with examples if plot related.
- [x] Documentation is sphinx and numpydoc compliant (the docs should [build](https://matplotlib.org/devel/documenting_mpl.html#building-the-docs) without error).
- [x] Conforms to Matplotlib style conventions (install `flake8-docstrings` and `pydocstyle<4` and run `flake8 --docstring-convention=all`).
- [n/a] New features have an entry in `doc/users/next_whats_new/` (follow instructions in README.rst there).
- [n/a] API changes documented in `doc/api/next_api_changes/` (follow instructions in README.rst there).

### CODE REVIEW

**Review by dstansby — APPROVED** [review]

File: doc/users/github_stats.rst [hunk 1]
```diff
@@ -3,138 +3,91 @@
...
-* jbhopkins
+* Greg Lucas
 * Jody Klymak
+* johnthagen
 * Jouni K. Seppänen
-* Lee Johnston
-* linchiwei123
-* Neilzon Viloria
+* Richard Sheridan
+* richardsheridan
```
Comment (jklymak) [inline doc/users/github_stats.rst:25]: this is a dupe?

Comment (jklymak) [inline doc/users/github_stats.rst:25]:
```suggestion
```

Comment (QuLogic) [reply doc/users/github_stats.rst:25]: This can be fixed, but the page it's on is automatically generated, so I wouldn't do it here.

Comment (jklymak) [reply doc/users/github_stats.rst:25]: Not important either way...

**Comment by tacaswell** [comment]
Given that the name duplication is auto-generated, I don't think we want to get into a game of whack-a-mole to fix it.

**Pull request merged**